frontend/src/views/admin/offertable/index.tsx

```diff
@@ -132,11 +134,12 @@ export function ConnectedOfferTable(props) {
     // are generated on-the-fly, memoize the result so we don't trigger unneeded re-renders.
     const columns = React.useMemo(() => {
         // Bind an `ApplicantCell` to a particular field
-        function generateApplicantCell(field) {
-            return (props) => (
+        function generateApplicantCell(field: any) {
+            return (props: any) => (
                 <ApplicantCell
                     field={field}
-                    upsertApplicant={(...args) =>
+                    upsertApplicant={(...args: any) =>
```

Comment:
We shouldn't need to destructure `args` here. We know the type of `upsertApplicant`.